docs/3-After-The-Offer/salary_and_negotiating.md

```diff
@@ -1,18 +1,56 @@
+The third time was far less than half of my current salary, but I managed 4 months worth of tuition! (Tuition is cheaper in Canada)
+
+When you're a student and the most money you've made is from distributing newspapers, the first offer you get can seem like a money. If one company pays an extra 1$/hour more than the other, that's 500$ over a summer. It feels like a pretty big difference. Almost a full iPhone.
+
+And maybe it is for you. Maybe you are struggling to pay off your living expenses and would like to support your family. Do what you need to do.
+
+For everyone else, keep in mind that you are doing an internship to learn, not to get rich. This isn't about how evil money is and all that. It's just that given that you are only working 3-4 months, the difference between your offers will be quite negligible in the long run in most cases. On the other hand, doing to a place with better learning opportunities will often unlock more options for future internships and full-time employment, some of which will pay more. And at that point, the difference will start to be significant, because you'll be working at that place for years. So even if money really is your end goal, think about it over the long run.
+
+Also, most humans are quite bad at estimating what will make them happy. You should read this [research on money and happiness](https://scholar.harvard.edu/files/danielgilbert/files/if-money-doesnt-make-you-happy.nov-12-20101.pdf).
+
+### Negotiating
+
+As mentioned above, internships are more about learning than making money. That being said, if you still intend to negotiate, then there's a few things to keep in mind.
+
+From the perspective of the company, there's a few reasons why they might not want to raise your offer:
+1. They want to hire interns are cheaply as possible
+2. They don't have the money
+3. They can hire someone just as good as you for the same amount of money
+4. They want all their interns to be paid the same to avoid resentment
+
+You should try to figure out what constraints they have to use the most effective approach in negotiating.
+
+It's difficult to do anything about (1) and (2). If you can demonstrate that you are significantly more productive, you might be able to make an economic argument that they'll get so much more work out of you that it's worth the slight increase in cost, but it will be hard.
+
+Case (3) will often be true at well-known, competitive companies. If you don't have a truly extraordinary accomplishment, negotiating might not be seen well.
+
+As a rule of thumb, if internship salary is publicly available for a company, (4) is likely to apply. You'll have a better chance of negotiating at smaller companies with fewer interns. Your negotiation is more likely to be successful if you can give them a story that would justify the salary disparity if other interns were to ever find out. This could include: "you have significantly more work experience", "you are senior", "you are a return intern".
```

Comment:
can also include "I was paid more at my last internship" or "my other offer is offering me more"